cmd/server/main.go

```diff
@@ -150,7 +151,8 @@ func (o *ProxyRunOptions) Flags() *pflag.FlagSet {
 	flags.UintVar(&o.agentPort, "agent-port", o.agentPort, "Port we listen for agent connections on.")
 	flags.UintVar(&o.adminPort, "admin-port", o.adminPort, "Port we listen for admin connections on.")
 	flags.UintVar(&o.healthPort, "health-port", o.healthPort, "Port we listen for health connections on.")
-	flags.DurationVar(&o.keepaliveTime, "keepalive-time", o.keepaliveTime, "Time for gRPC server keepalive.")
+	flags.DurationVar(&o.agentKeepaliveTime, "keepalive-time", o.agentKeepaliveTime, "Time for gRPC server keepalive.")
```

Comment:
grpc server/grpc master are unlikely to mean much to the user. Can we use agent/server? That corresponds on the port descriptions we have above.